### PR TITLE
Feat: Use default config for testnets

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Simplify dfx.json

# Changes

* Use default config for `FETCH_ROOT_KEY` and `FEATURE_FLAGS` in most testnets.

# Tests

Not applicable.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@dskloetd @bitdivine please review

File: dfx.json [hunk 1]
```diff
@@ -362,183 +289,79 @@
     },
     "https___benchmarklarge_testnet_dfinity_network": {
       "config": {
-        "FETCH_ROOT_KEY": true,
-        "HOST": "https://benchmarklarge.testnet.dfinity.network",
-        "FEATURE_FLAGS": {
-          "ENABLE_SNS_2": true,
-          "ENABLE_SNS_VOTING": true,
-          "ENABLE_SNS_AGGREGATOR": true,
-          "ENABLE_CKBTC": false,
```
Comment (dskloetd) [inline dfx.json:371]: This is not the default value so you need to override it, right?

File: dfx.json [hunk 2]
```diff
@@ -362,183 +289,79 @@
...
     },
     "https___large01_testnet_dfinity_network": {
       "config": {
-        "FETCH_ROOT_KEY": true,
-        "HOST": "https://large01.testnet.dfinity.network",
-        "FEATURE_FLAGS": {
-          "ENABLE_SNS_2": true,
-          "ENABLE_SNS_VOTING": true,
-          "ENABLE_SNS_AGGREGATOR": true,
-          "ENABLE_CKBTC": false,
```
Comment (dskloetd) [inline dfx.json:385]: This is not the default value so you need to override it, right?
And many more below. Please check that you only remove values that are the default value.

Comment (lmuntaner) [reply dfx.json:385]: Ditto

**Comment by dskloetd** [comment]
This changes lots of values. I think it would be better to be explicit about all the changes that this causes.
But I never deploy testnets so if someone else thinks this is fine, I'm happy to stand back while someone else approves.
But personally I would separate changing the configurations from using the default configuration.

**Comment by lmuntaner** [comment]
@mstrasinskis please review

**Review by mstrasinskis — APPROVED** [review]
LGTM, thanks!

**Review by bitdivine — APPROVED** [review]
Perfect, thank you!

**Pull request merged**